src/utils/common/isPollingFrequencyMet.ts

```diff
@@ -0,0 +1,6 @@
+// returns true if the polling frequency has been reached
+// returns false if the polling frequency has not been reached
+export function isPollingFrequencyMet(frequency: number, lastHeight: number, currentHeight: number): boolean {
+  // console.log(`frequency: ${frequency} lastHeight: ${lastHeight} currentHeight: ${currentHeight} next: ${lastHeight + frequency}`);
+  return lastHeight + frequency <= currentHeight ? true : false;
```

Comment:
what you have here is very explicit and works perfectly. if you don't already know, you can also skip the ternary  and simply `return lastHeight + frequency <= currentHeight`. this is because you're returning true when the ternary is true and false when it's false eg: `condition ? true : false`

this is all for fyi for you. i don't see why this would need to change in the code :)

Reply:
great point, I went ahead and changed it because it does look kinda weird using the ternary like that.